src/asmcnc/apps/drywall_cutter_app/config/config_loader.py

```diff
@@ -3,8 +3,8 @@
 import config_classes
 import inspect
 
-configurations_dir = 'asmcnc/apps/drywall_cutter_app/config/configurations'
-cutters_dir = 'asmcnc/apps/drywall_cutter_app/config/cutters'
+configurations_dir = './asmcnc/apps/drywall_cutter_app/config/configurations'
+cutters_dir = './asmcnc/apps/drywall_cutter_app/config/cutters'
 
```

Comment:
These paths cause issues with running unit test, but I've fixed this on another branch.